src/com/amazon/ionelement/api/IonUtils.kt

```diff
@@ -54,7 +53,7 @@ fun IonValue.toIonElement(): IonElement =
     }
 
 /** Throws an [IonElectrolyteException], including the [IonLocation] (if available). */
-fun ionError(blame: IonElement, description: String): Nothing {
+fun ionError(blame: Element, description: String): Nothing {
     ionError(blame.metas, description)
 }
 
```

Comment:
just a reminder that this is the last file within the public api and you can stop reviewing now.